apps/server/swagger/dictionary-api.yml

```diff
@@ -23,6 +23,9 @@
                 type: string
                 description: A matching Dictionary Version defined on Dictionary Manager (Lectern)
                 required: true
+              defaultCentricEntity:
+                type: string
+                description: The default centric entity name
```

Comment:
Adding new input parameter on Dictionary Registration endpoint. This is value is used to retrieve compound records